tests/integration/operator/switch_cr_test.go

```diff
@@ -403,7 +403,7 @@ func compareInClusterAndGeneratedResources(t framework.TestContext, cs cluster.C
 				_, err = cs.PolicyV1beta1().PodDisruptionBudgets(ns).Get(context.TODO(), name,
 					kubeApiMeta.GetOptions{})
```

Comment:
(comment location is random)

I am seeing warning on install with 1.19:
```
warn    installer       retrieving resources to prune type autoscaling/v2, Kind=HorizontalPodAutoscaler: no matches for kind "HorizontalPodAutoscaler" in version "autoscaling/v2"
```

Reply:
it is expected on pre 1.23 versions, I thought of updating the pruning logic to check for versions as well but may prefer to do it in a follow up PR instead

Reply:
I don't think we want this warning to trigger on all installs of <1.23. Most users will probably think something is broken

Reply:
done